cobra.go

```diff
@@ -40,14 +40,23 @@ var templateFuncs = template.FuncMap{
 
 var initializers []func()
 
+const (
+	DefaultPrefixMatching  = false
+	DefaultCommandSorting  = true
+	DefaultCaseInsensitive = false
```

Comment:
Are you imagining a scenario where a program would like to access these?
I was thinking a program may want to use the same default as Cobra for case-sensitivity of other things; however, it wouldn't be the constant they should use but the value set in `cobra. EnableCaseInsensitive`.

So, if these constants don't need to be exported, let's make them private.

Reply:
Very true Marc! +1 - I don't expect users to need these constants since they delineate internal default behavior

Reply:
Agree. Just wanted to make those constants so they can be used in tests where toggling back to default configuration . Casting to private :)